database/layer/admin-tenant.js

```diff
@@ -79,3 +74,46 @@ async function deleteTenant (tenantName, issClaim) {
     return `Delete failed: ${error.message}`
   }
 }
+
+/**
+ * Get tenants by optional filters
+ *
+ * @param {Object} filters - The filters to apply
+ * @param {number} [filters.key] - The tenant key
+ * @param {string} [filters.name] - The name of tenant
+ * @param {string} [filters.iss_claim] - The iss_claim of tenant
+ * @param {string} [sort='name'] - The column to sort by
+ * @param {string} [order='desc'] - The order of sorting (asc or desc)
+ * @returns {Promise<Array<Object>>} - A promise that resolves to an array of tenants
+ */
+const getTenants = async (filters = {}, sort = 'key', order = 'asc') => {
```

Comment:
@k1ch  I would have the default sort order be `desc`. 
Generally users want to see the most recent X of something instead of the oldest.